### PR TITLE
remove default admin data, fix editUser command

### PR DESCRIPTION
**Pull request by MariyaSin**

### CODE REVIEW

File: src/UserService.Business/Commands/User/EditUserCommand.cs [hunk 1]
```diff
@@ -263,41 +320,46 @@ public OperationResultResponse<bool> Execute(Guid userId, JsonPatchDocument<Edit
...
+
+            if (isActiveOperation != null)
+            {
+                bool dbUserIsActive = _userRepository.Get(userId).IsActive;
+
+                if (bool.Parse(isActiveOperation.value.ToString()) && !dbUserIsActive)
+                {
+                    string password = _generatePassword.Execute();
+
+                    _userRepository.CreatePending(new() { UserId = userId, Password = password });
```
Comment (BelehovEgor) [inline src/UserService.Business/Commands/User/EditUserCommand.cs:353]: why we create new password if user has old password

File: src/UserService.Data.Provider.MsSql.Ef/Migrations/20201011111059_ReworkDbModels.cs [hunk 2]
```diff
@@ -92,11 +34,6 @@ protected override void Down(MigrationBuilder migrationBuilder)
                 keyColumn: "Id",
```
Comment (BelehovEgor) [inline src/UserService.Data.Provider.MsSql.Ef/Migrations/20201011111059_ReworkDbModels.cs:34]: why did not you delete this?

File: src/UserService.Data/UserCredentialsRepository.cs [hunk 3]
```diff
@@ -105,12 +105,31 @@ public bool Edit(DbUserCredentials userCredentials)
...
             return dbUserCredentials.Id;
         }
 
+        public bool Remove(Guid userId)
+        {
+            DbUserCredentials credentials = _provider.UserCredentials.FirstOrDefault(c => c.UserId == userId);
+
+            if (credentials != null)
+            {
+                _provider.UserCredentials.Remove(credentials);
```
Comment (BelehovEgor) [inline src/UserService.Data/UserCredentialsRepository.cs:125]: we talk about deactivated, not removing

**Review by BelehovEgor — CHANGES_REQUESTED** [review]

File: src/UserService.Business/Commands/User/EditUserCommand.cs (hunk 1 above)
Comment (MariyaSin) [reply src/UserService.Business/Commands/User/EditUserCommand.cs:353]: it is not safe to use old password

File: src/UserService.Data.Provider.MsSql.Ef/Migrations/20201011111059_ReworkDbModels.cs (hunk 2 above)
Comment (MariyaSin) [reply src/UserService.Data.Provider.MsSql.Ef/Migrations/20201011111059_ReworkDbModels.cs:34]: We do not need this data in release

File: src/UserService.Data/UserCredentialsRepository.cs (hunk 3 above)
Comment (MariyaSin) [reply src/UserService.Data/UserCredentialsRepository.cs:125]: We deactivated user, credentials we can remove

File: src/UserService.Data.Provider.MsSql.Ef/Migrations/20201011111059_ReworkDbModels.cs (hunk 2 above)
Comment (MariyaSin) [reply src/UserService.Data.Provider.MsSql.Ef/Migrations/20201011111059_ReworkDbModels.cs]: done

**Review by BelehovEgor — APPROVED** [review]

File: src/UserService.Data/UserCredentialsRepository.cs [hunk 4]
```diff
@@ -105,12 +108,32 @@ public bool Edit(DbUserCredentials userCredentials)
...
+                throw new ArgumentNullException(nameof(dbUserCredentials));
+            }
+
             _provider.UserCredentials.Add(dbUserCredentials);
             _provider.Save();
 
             return dbUserCredentials.Id;
         }
 
+        public void SwitchActiveStatus(Guid userId, bool status)
```
Comment (djino97) [inline src/UserService.Data/UserCredentialsRepository.cs:122]: status => isActiveStatus

**Review by djino97 — APPROVED** [review]

File: src/UserService.Data/UserCredentialsRepository.cs (hunk 4 above)
Comment (MariyaSin) [reply src/UserService.Data/UserCredentialsRepository.cs:122]: done

File: src/UserService.Business/Commands/User/EditUserCommand.cs [hunk 5]
```diff
@@ -78,24 +81,22 @@ private bool ChangeUserPosition(Guid positionId, Guid userId, List<string> error
...
-                }
+                    _logger.LogWarning(logMessage, positionId, userId);
 
-                _logger.LogWarning(logMessage, positionId, userId);
+                    errors.Add(errorMessage);
+                }
             }
             catch (Exception exc)
             {
                 _logger.LogWarning(exc, logMessage, positionId, userId);
```
[the remark this answers is not in the document]
Comment (MariyaSin) [reply src/UserService.Business/Commands/User/EditUserCommand.cs]: done

**Review by borovHokage — APPROVED** [review]

**Pull request merged**